src/embind/embind.js

```diff
@@ -439,6 +439,7 @@ var LibraryEmbind = {
 
   _embind_register_bool__deps: [
     '$getShiftFromSize', '$readLatin1String', '$registerType'],
+  _embind_register_bool__sig: 'vpppii',
```

Comment:
Here's where I start asking ~~dumb~~ basic questions about JS libs.
what happens when there's no signature on a JS lib function? What does adding them here accomplish?

Reply:
There are a couple of different uses for these signatures. 

In this case we need the signature so that we can generate automatic BigInt<->Number conversion wrapper at the boundary.  The `p` here means pointer and is used for pointer/size_t things that we know assume won't exceed 53-bits..

Reply:
Without these the JS code recieves BitInt values when running with MEMORY64.. which most of our JS code is not setup to receive.